plugins/admission/pkg/models/models.go

```diff
@@ -35,7 +35,8 @@ type Configuration struct {
 		CustomChecks         []opa.OPACustomCheck
 		CustomCheckInstances []opa.CheckSetting
 	}
-	Polaris *polarisconfiguration.Configuration
+	Polaris               *polarisconfiguration.Configuration
+	IgnoreServicesAccount []string
```

Comment:
Shouldn't it be called `IgnoreUsernames`?

Reply:
Not sure which one is more clear

Reply:
Yeah I think `Usernames` is better--service accounts are a type of user.

Reply:
I'm not sure this should go in the config. That gets stored in the API IIRC. I was thinking it would be a flag in the helm chart (e.g. an environment variable or command-line flag).

Maybe this is better though? I'm not sure. Lemme think about it